recipes/pycudasirecon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pycudasirecon" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tlambert03/pycudasirecon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 7c51a06d2789212df029506c7d091c589edecfb0395aa8ef3534b3fb578fb9e3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - pydantic
+    - cudasirecon
+    - tifffile
+    - typing_extensions
+
+test:
+  imports:
+    - pycudasirecon
```

Comment:
```suggestion
    - pycudasirecon
    - pip check
  requires:
    - pip
```

Reply:
Merged, but broke it.  Should be a command no?

Reply:
Yup sorry about that 😅

Reply:
no worries.  thanks for the tip!